packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -489,6 +489,26 @@ contract SphinxUtils is SphinxConstants, StdUtils {
         return config;
     }
 
+    function assertValidOwners(SphinxConfig memory _config) private pure {
+        require(
+            _config.owners.length > 0,
+            "Sphinx: You must have at least one owner in your 'sphinxConfig.owners' array before calling this function."
+        );
+
+        for (uint256 i = 0; i < _config.owners.length; i++) {
+            // We prevent the owners from being either address(0) or address(0x1)
+            // address(0x1) is the SENTINAL_ADDRESS which is a special address used
+            // by Safe to handle their owner implementation. We check for it here for
+            // completeness, but it's unlikely the user would ever actually use that
+            // value so we don't mention it in the error message.
+            if (_config.owners[i] == address(0) || _config.owners[i] == address(0x1)) {
+                revert(
+                    "Sphinx: Detected owner that is address(0). You are not allowed to use the zero address as a Gnosis Safe owner"
+                );
```

Comment:
nit: I could see this restriction being annoying to users who just want to use temporary placeholder addresses as their owners. It may be nice to direct their annoyance to Gnosis Safe since this isn't our restriction. The current error message implies that it's our restriction, or at least doesn't make it clear that it's Gnosis Safe's.

The second sentence in the error message could be something like, "Gnosis Safe prevents you from using this address as an owner"